docs/platforms/apple/common/user-feedback/index.mdx

```diff
@@ -1,15 +1,109 @@
+The User Feedback widget allows users to submit feedback from anywhere inside your application.
 
-<PlatformContent includePath="user-feedback/sdk-api-example/" />
+### Pre-requisites
+
+Ensure that your project is set up with Sentry and that you have added the Sentry Cocoa SDK to your project.
+
+### Installation
+
+1. Install the Sentry Cocoa SDK using CocoaPods, Carthage, or Swift Package Manager.
+2. Ensure you are using version 8.35.0 or above of the SDK to access the latest features.
+
+### Set Up
+
+To start using the User Feedback widget in your Cocoa application, enable the feedback integration and provide an optional configuration in your SentryOptions when starting the SDK:
+
+```swift
+ SentrySDK.start { options in
+     options.configureUserFeedback = { config in
+          config.onSubmitSuccess = { data in
+               print("Feedback submitted successfully: \(data)")
+          }
+          config.onSubmitError = { error in
+               print("Failed to submit feedback: \(error)")
+          }
+     }
+ }
+```
+
+This setup will insert the widget into your app's view hierarchy. By default, it appears in the bottom trailing corner of the screen, but you can fully customize its appearance and behavior.
+
+### Customization
+
+You can customize the widget by setting various properties in the instance of `SentryUserFeedbackConfiguration` provided by `SentryOptions.configureUserFeedback`:
+
+```swift
+let widgetConfig = SentryUserFeedbackWidgetConfiguration()
+widgetConfig.triggerLabel = "Feedback"
+widgetConfig.triggerAccessibilityLabel = "Submit Feedback"
+
+// Form customization
+let formConfig = SentryUserFeedbackFormConfiguration()
+formConfig.showName = false
+formConfig.isNameRequired = false
+formConfig.enableScreenshot = true
+formConfig.formTitle = "Report an Issue"
+formConfig.submitButtonLabel = "Submit"
+formConfig.cancelButtonLabel = "Cancel"
+
+let feedbackConfig = SentryUserFeedbackConfiguration { config in
+    config.widgetConfig = widgetConfig
+    config.uiFormConfig = formConfig
+}
+
+let feedbackIntegration = SentryUserFeedbackIntegration(configuration: feedbackConfig)
+feedbackIntegration.createWidget()
+```
+
+This configuration customizes the widget's button text, form fields, and whether the user can attach screenshots.
+
+### Session Replay
+
+The User Feedback widget integrates seamlessly with Session Replay. When the widget is opened, the Replay SDK buffers up to 30 seconds of the user's session. If feedback is submitted, this replay is sent along with the feedback, allowing you to view both the feedback and the user's actions leading up to the feedback submission.
```

Comment:
@bruno-garcia @brustolin I'm not sure if the 30 second thing checks out or not. All I'm doing is attaching any available session replay id when I package the envelope. Let me know if we need to look into this some more.

Reply:
The expectation is that if  `replay_id` is in the scope, we have an active replay or, one is buffering.
Of the widget is popped up, we should sample that reply (so flush the buffer, however long it is). The buffer could be 10 seconds for example (say the app just ran for 10 seconds).

@armcknight is this the behavior we get out of the widget today if it pops up? (we capture the current buffer/replay?)
@billyvg does this align with Replay + Feedback Widget behavior on Web?
@philprime does this align with the Replay behavior on iOS?